actions/next-semver/dist/index.js

```diff
@@ -0,0 +1,2878 @@
+/******/ (() => { // webpackBootstrap
```

Comment:
Do we need the `actions/next-semver/dist` folder to be checked in?

Reply:
yes, the action is compiled to a single runnable js file for any consuming workflows. We could probably introduce some sort of pre step in the action.yml to compile at run but this would slow the action down considerably